proto/minder/v1/minder.proto

```diff
@@ -2313,27 +2313,42 @@ message RoleAssignment {
     // subject is the subject to which the role is assigned.
     string subject = 2;
     // display_name is the display name of the subject.
-    string display_name = 5 [(buf.validate.field).string = {
-        pattern: "[A-Za-z][- [:word:]\\[\\]\\(\\)]*",
-        max_len: 200,
-    }, (buf.validate.field).ignore_empty = true];
+    string display_name = 5 [
+        (buf.validate.field).string = {
+            pattern: "[A-Za-z][- [:word:]\\[\\]\\(\\)]*",
```

Comment:
I'm not sure you want names `[ to include brackets ]`

Reply:
I should test regex before using them 😃 This is the regex you suggested in the previous PR but I admit I did not verified it. Let me check 👍

Reply:
That regex allows for capital or lowercase, followed by `-`, space, A-Z, a-z, 0-9, `_`, square brackets, and parenthesis.  But I'm not sure that's what we want for the `display_name` field, which seems to be taken from the GitHub username / slug, which may be closer to `[-[:word:]]+`, or `[A-Za-z][-[:word:]]*`.

Reply:
I think we also allow the user to set this, so I guess [ ] can happen? Of course if we want to allow for it.